pkg/kube/exec.go

```diff
@@ -147,7 +148,7 @@ func execute(method string, url *url.URL, config *restclient.Config, stdin io.Re
 	if err != nil {
 		return err
 	}
-	return exec.Stream(remotecommand.StreamOptions{
+	return exec.StreamWithContext(context.TODO(), remotecommand.StreamOptions{
```

Comment:
We should propagate context from parent functions and pass it here.

Reply:
we are not getting any context from parent function. if we decide to provide one, it might break things. hence created a new context here

Reply:
We can do it in a follow up 👍🏼

Reply:
@shahpratikr Please file a JIRA for that.

Reply:
Added link in a comment